specs/010-synapse-kratos-oidc/tests/T020c-e2e-oauth2-flow-test.md

```
@@ -75,20 +76,24 @@ User (Element Client)
    - **Expected**: Redirect to Synapse SSO endpoint
 
 2. **Synapse → Hydra Redirect** (FR-002):
+
    ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)